Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -36,6 +36,12 @@ def IsComplete (s : Set α) :=
   ∀ f, Cauchy f → f ≤ 𝓟 s → ∃ x ∈ s, f ≤ 𝓝 x
 #align is_complete IsComplete
 
+/-- A version of `lim` that deduces `Nonempty` from `Cauchy`.
+
+This is an abbreviation, so that we do not need to repeat all API lemmas. -/
+protected noncomputable abbrev Cauchy.lim (f : Filter α) (hf : Cauchy f) : α :=
+  @lim _ _ hf.1.nonempty f
```

Comment:
I think this would be cleaner with a `haveI` (the underlying term should be the same, `I` stands for inlined)